packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -11,9 +11,9 @@
 
 * Changed
   * Updated screenshots with elements that are out of bound
-
 * Changed
   * Updated Jest command
+  * Updated Swedish translations.
```

Comment:
This should be in the moved up to the unreleased section. Looks like there are two Changed sections here. Can you combine them also?